lib/kinesis_client/stream/coordinator.ex

```diff
@@ -116,12 +116,21 @@ defmodule KinesisClient.Stream.Coordinator do
 
         {:noreply, %{state | shard_graph: shard_graph, shard_ref_map: map}}
 
-      other ->
-        Logger.info("Stream is not in active state, sleeping... Stream state: #{inspect(other)}")
```

Comment:
Previously the Coordinator would retry if the stream returns info that it's an inactive state. I've changed this so such a case will immediately error. We should consider an attempt to connect to an inactive stream a fatal error for our uses.

Reply:
Good for now, we _might_ need to revisit this when we do shard splits/merges.

Reply:
Oh absolutely!